stripper/ze_paranoid_rezurrection_csgo1.cfg

```diff
@@ -1,50 +1,310 @@
-; "ze_paranoid_rezurrection_csgo1" config, made by "Berke" "STEAM_1:0:95142811", version 1.
+// "ze_paranoid_rezurrection_csgo1" config, made by "Berke" "STEAM_1:0:95142811", version 2.
 
-; Changes from "GFL".
-; Fix "game_player_equip" dropping weapons and fix kevlar buying.
+// Delete low buy time, extend height of spawn trigger and fix zombies not getting teleported.
 modify:
 {
 	match:
 	{
-		"classname" "game_player_equip"
-		"targetname" "darP90"
+		"classname" "logic_auto"
+	}
+	insert:
+	{
+		"OnMapSpawn" "protect_player,AddOutput,solid 2,,1"
```

Comment:
You're missing the delay parameter on many outputs in here such as this one.

Reply:
Not needed, when left empty it defaults to 0.

Reply:
Should be fine then https://github.com/perilouswithadollarsign/cstrike15_src/blob/master/game/server/cbase.cpp#L117, still not really a fan of this though.